src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -40515,6 +40515,18 @@ unit:REV-PER-MIN
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Revolution per Minute"@en .
 
+unit:REV-PER-MIN-SEC
+  a qudt:Unit ;
+  dcterms:description "Revolution per Minute per Second is a unit for Angular Acceleration (sometimes called Change of RPM, crpm)"^^qudt:LatexString ;
+  qudt:conversionMultiplier 376.9911186 ;
```

Comment:
As you mentioned in the issue, you calculated this from the REV-PER-SEC2 unit. However, I think you should divide that multiplier by 60 rather than multiply it by 60, right?

Reply:
right you are!